packages/docs/components/text-input.md

```diff
@@ -597,15 +597,15 @@ Out of the box, Odyssey supports input types for `text`, `email`, `search`, `tel
 
 <Description>
 
-  Because of the current inability to ensure consistent validation behavior across browsers, we're using the `[data-invalid]` attribute to indicate this state.
+  Because of the current inability to ensure consistent validation behavior across browsers, we're using the `.is-ods-input-invalid` class to indicate this state.
```

Comment:
Is this something we've experience ourselves or just know from research? It is unfortunate that we need to apply multiple ways to solve a problem (pseudo-class and class) because it adds complexity into our CSS

Reply:
The main use cases for this are:
- support for for custom form components without native validation
- support for using Odyssey CSS with Courage (or other feature team) validation
- support for irregular browsers

AFAIK, we have since dropped support for browsers (<IE11) that don't validate as expected. My ideal would be to utilize `:invalid` and `:valid` as normal, but that may not work for all cases.

Another option here would be to utilize `aria-invalid="true"` as our error selector. That would give us an extra a11y affordance, but would not be applied in non-React scenarios.